evalutils/templates/processor/hooks/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
coding is not needed - this is Python 3 only

Reply:
ok